src/utils/colorlib.ts

```diff
@@ -1,4 +1,4 @@
-import { Color } from "./Color";
+import { Color } from "./color";
```

Comment:
Please add the ".js" suffix at the end of the line.

Reply:
done